src/main/resources/application.conf

```diff
@@ -2,3 +2,17 @@ akka {
   loglevel = INFO
   log-dead-letters-during-shutdown = off
 }
+
+# The server/client pipeline configuration for pipline flow
```

Comment:
Just to not confuse (as this setting can be delivered by a library, too) can we change this to

```scala
# The default server/client pipeline. This can be overridden by each service and client in the application.
```